### ISSUE TITLE
Missing border of an Image when you export from docx to rtf, convert to pdf works fine

### ISSUE THREAD

**Issue by WEONDO** [CollaboraOnline/online#8835]
### Describe the Bug
When you convert an docx file with inside images with borders, the border exist not any more when you convert it to rtf. The convert to pdf works fine.

### Steps to Reproduce
URL [convertto ](https://localhost:9980/cool/convert-to/rtf)
### Expected Behavior
After the convert from docx to rtf, the borders must also exist like in the docx document

### Actual Behavior
After the convert, the borders are not exist anymore

### Desktop
 - **Collabora version:** 23.05.9-3
 - **OS and version:** Ubuntu 22.04.01
 - **Browser and version:** every browser

**Comment by WEONDO** [comment]
The same problem exists also in 23.05.10-1

**Comment by timur-g** [comment]
That is true and still so with version 25.04. Both opening image with border from RTF and saving to RTF do not work.
But, the issue comes from Core and I am reporting with TDF and closing this one.
